src/convert.jl

```diff
@@ -16,6 +16,10 @@ function convert(::Type{Polymake.pm_Integer},int::T) where T <: Union{Int128,Int
     return Polymake.pm_Integer(int)
 end
 
+function convert(::Type{Polymake.pm_Set}, A::Vector{T}) where T<:Signed
```

Comment:
Is this just WIP? Otherwise, shouldn't the `T` be inherited from `Integer`?

Reply:
All of this is a very first-try effort ;-)

`Signed` is the type of all machine-style signed integers. I don't make any promises of the sets of arbitrary sized ints, at least for now. And it's a placeholder, until parametrized set lands anyway.

Reply:
Okay, then it is perfect!